Automattic/wp-calypso
# Fix/jetpack obsolete gates

This PR removes a couple of obsolete feature gates that should be always open for all the cases now.

how to test
========
1. Go to http://calypso.localhost:3000/jetpack/connect/store and check you can see the personal plan there
2. Go to http://calypso.localhost:3000/jetpack/connect and check you can connect a jetpack site there ;)
